### ISSUE TITLE
Button to add / read comments

### ISSUE THREAD

**Issue by cdipe** [brickfield/moodle-mod_board#94]
Users don't understand that they can click a post to add / read comments. A button would be great!

**Comment by learningtechnologyservices** [comment]
@cdipe yes, we are looking into implementing this feature for the comments section, thanks for your feedback!

**Comment by learningtechnologyservices** [comment]
Hi @cdipe, many thanks for using Board, and your comments suggestion should now be available with [our latest release here](https://github.com/brickfield/moodle-mod_board/commit/397a995cc1066e9be7f5b02fe33e2164ec3accce) and updated on our [Moodle Plugins database page](https://moodle.org/plugins/mod_board) also!

**Comment by cdipe** [comment]
Great, thank you!
